Makefile

```diff
@@ -951,7 +951,64 @@ ifeq "$(OPENMP)" "true"
 endif
 
 
-pio_test:
+openacc_test : 
+ifeq "$(OPENACC)" "true"
+	@#
+	@# First ensure that both FFLAGS_ACC and CFLAGS_ACC are not blank
+	@# If these are not set for a target, then OpenACC most likely cannot compile
+	@#
+	@echo "Checking if FFLAGS_ACC and CFLAGS_ACC are defined for [$(BUILD_TARGET)]..."
+	@if ([ -z "$(FFLAGS_ACC)" ] && [ -z "$(CFLAGS_ACC)" ]); then \
+	   (echo "************ ERROR ************"; \
+	    echo "OPENACC=true was specified, but [$(BUILD_TARGET)] build target does not seem to support OpenACC:"; \
+	    echo "    FFLAGS_ACC and CFLAGS_ACC are both undefined for [$(BUILD_TARGET)] in the top-level Makefile."; \
+	    echo "Please set these variables to appropriate OpenACC compilation flags in the [$(BUILD_TARGET)] target to enable OpenACC support."; \
+	    echo "************ ERROR ************"; exit 1) \
+	else \
+	   echo "=> FFLAGS_ACC or CFLAGS_ACC defined"; \
+	fi
+
+	@#
+	@# Create test C and FORTRAN programs that look for OpenACC header file and parallelize a loop
+	@#
+	@printf '#include <openacc.h>\nint main(){\n int CORES=acc_get_num_cores(); int i,n=0;\n #pragma acc kernels\n for (i=0; i<10; i++); n=n+i;\n return 0;}\n' > openacc.c
+	@printf 'program openacc1\n use openacc; external acc_get_num_cores\n integer :: i,n=0,CORES=0\n CORES=acc_get_num_cores()\n !$$acc kernels\n DO i=0,10; n=n+i; END DO\n !$$acc end kernels\nend program\n' > openacc.f90
```

Comment:
I did the one-liner to try and limit the number of lines added to the Makefile, but this version would let someone inspect the test programs without having to remove the `rm` instructions. My preference leans towards the one-liner since I'm still new to `sed`.

Reply:
Push to a22bee0 should resolve this with multi-line test programs